Added the 2 new columns in pokestop needed for MAD

AboveColin opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
<!--  NOTE: In order to check code style locally and avoid having your build rejected by Travis, -->
<!--  run the following commands before you commit: `flake8 .` and `npm run lint`. Fix any -->
<!--  issues they point out. Note also that flake's NOQA is disabled on Travis. -->
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

nakratz commented:
Should the Pokestop Model be updated with the variables as well.
https://github.com/cecpk/OSM-Rocketmap/blob/c0e40510572655bcaa93429db13f6554a581a741/pogom/models.py#L361

name = Utf8mb4CharField(null=True, default=NULL, max_length=128)
image = Utf8mb4CharField(null=True, default=NULL, max_length=255)

AboveColin commented:
> I deleted my last comment as I didn't realize that the image and name were already defined in the pokestop model. To be consistent they should be changed to the following to match your alter table info.
> 
> name = Utf8mb4CharField(null=True, default=NULL, max_length=128)
> [OSM-Rocketmap/pogom/models.py](https://github.com/cecpk/OSM-Rocketmap/blob/55d94d11c5cfa9303bbf44f17b4c448cc802c6d5/pogom/models.py#L354)
> 
> Line 354 in [55d94d1](/cecpk/OSM-Rocketmap/commit/55d94d11c5cfa9303bbf44f17b4c448cc802c6d5)
> 
>  name = Utf8mb4CharField(max_length=200) 
> 
> image = Utf8mb4CharField(null=True, default=NULL, max_length=255)
> [OSM-Rocketmap/pogom/models.py](https://github.com/cecpk/OSM-Rocketmap/blob/55d94d11c5cfa9303bbf44f17b4c448cc802c6d5/pogom/models.py#L355)
> 
> Line 355 in [55d94d1](/cecpk/OSM-Rocketmap/commit/55d94d11c5cfa9303bbf44f17b4c448cc802c6d5)
> 
>  image = Utf8mb4CharField(max_length=200)

Updated 👍
